systemtest/src/test/java/io/strimzi/systemtest/kafka/dynamicconfiguration/DynamicConfST.java

```diff
@@ -54,13 +60,14 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
-/**
- * DynamicConfST is responsible for verify that if we change dynamic Kafka configuration it will not
- * trigger rolling update.
- * Isolated -> for each test case we have different configuration of Kafka resource
- */
 @Tag(REGRESSION)
 @Tag(DYNAMIC_CONFIGURATION)
+@SuiteDoc(
+    description = @Desc("DynamicConfST is responsible for verifying that changes in dynamic Kafka configuration do not trigger a rolling update."),
+    beforeTestSteps = {
+        @Step(value = "Deploy the Cluster Operator.", expected = "Cluster Operator is installed successfully.")
```

Comment:
I think you use somewhere `Cluster Operator` and somewhere `cluster operator`, guess it would be good to unite it

Reply:
moved to cluster operator.